src/components/Navbar/Navbar/style.scss

```diff
@@ -0,0 +1,150 @@
+@import '../../../scss/variables';
+@import '../../../scss/base';
+
+$white: #ffffff;
+*,
```

Comment:
you can move section 5 - 11. we already have box-sizing integrated with this app. we wouldn't want to set something to all * inside of nav bar style.